contrib/platform/src/com/sun/jna/platform/linux/LibC.java

```diff
@@ -38,7 +38,7 @@
 /**
  * LibC structures and functions unique to Linux
  */
-public interface LibC extends LibCAPI, Library {
+public interface LibC extends LibCAPI, ErrNo, Fcntl, Mman, Library {
```

Comment:
Because I saw it in the past! :)

I suppose it allows access to the other members without having to call the other class, but actually using `LibC.EPERM` is frowned upon anyway, and `import static` is superior to both, I think.  So I'll go ahead and remove this.